pkg/config/operator/operator.go

```diff
@@ -53,3 +61,27 @@ Issue created by Allstar. See https://github.com/ossf/allstar/ for more informat
 // NoticePingDuration is the duration to wait between pinging notice actions,
 // such as updating a GitHub issue.
 const NoticePingDuration = (24 * time.Hour)
+
+var osGetenv func(string) string
+
+func init() {
+	osGetenv = os.Getenv
+	setVars()
+}
+
+func setVars() {
+	appIDs := osGetenv("APP_ID")
```

Comment:
Consider using `os.LookupEnv` which can tell you whether this var was set at all.

Reply:
Cool, In this case I would take the same action if the var is empty or not set, so I'll leave it. Thanks for pointing that one out, It will come in handy in the future.